src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -102,6 +112,33 @@ void OutgoingMigration::SyncFb() {
     migration->WaitForSnapshotFinished();
   }
   VLOG(1) << "Migrations snapshot is finihed";
+
```

Comment:
Also I assume we want to check the context here for cancellation?

Reply:
It's a future)